### DISCUSSION TITLE
KEDA Chart v2.10.2

### DISCUSSION THREAD

**Discussion by JorTurFer** [kedacore/charts#429] [Announcements]
Chart: v2.10.2 | App: v2.10.1

## TL;DR

```shell
helm repo add kedacore https://kedacore.github.io/charts
helm repo update
helm install keda kedacore/keda --version 2.10.2 -n keda --create-namespace
```

## What is new?

- Support for KEDA v2.10.1 ([release notes](https://github.com/kedacore/keda/releases/tag/v2.10.1))
- Handle custom cluster-domain values (#399- Contributed by @lucchmielowski 🎉)
- Allow setting Certificate secretTemplate (#426- Contributed by @pharaujo 🎉)
- Enable `readOnlyRootFilesystem: true` for metrics server (#402- Contributed by @JorTurFer 🎉)

### Fixes / Changes

- Fix extraVolumes and extraVolumeMounts indentation (#411 - Contributed by @joebowbeer 🎉)

### Breaking Changes

None.

## What's Changed
* feat: Handle custom cluster-domain values by @lucchmielowski in https://github.com/kedacore/charts/pull/399
* Fix extraVolumes and extraVolumeMounts indentation by @joebowbeer in https://github.com/kedacore/charts/pull/411
* chore: enable `readOnlyRootFilesystem: true` for metrics server by @JorTurFer in https://github.com/kedacore/charts/pull/412
* chore(keda): test KEDA chart in other namespaces with/without cert-manager by @JorTurFer in https://github.com/kedacore/charts/pull/414
* fix: Use proper stage names by @tomkerkhove in https://github.com/kedacore/charts/pull/416
* fix: Use proper stage names by @tomkerkhove in https://github.com/kedacore/charts/pull/417
* fix: adj indent of extraVolumes & VolumeMounts in 14-keda-deployment by @iggyshu in https://github.com/kedacore/charts/pull/419
* ci: http add on workflow now installs appropriate kedacore/keda chart for 1.23 clusters by @chrisjohnson00 in https://github.com/kedacore/charts/pull/423
* Add resource limits and requests for the interceptor pod component by @chrisjohnson00 in https://github.com/kedacore/charts/pull/421
* Allow setting Certificate secretTemplate by @pharaujo in https://github.com/kedacore/charts/pull/426
* chore: Release KEDA Chart v2.10.2 (KEDA v2.10.1) by @JorTurFer in https://github.com/kedacore/charts/pull/428

## New Contributors
* @lucchmielowski made their first contribution in https://github.com/kedacore/charts/pull/399
* @iggyshu made their first contribution in https://github.com/kedacore/charts/pull/419
* @chrisjohnson00 made their first contribution in https://github.com/kedacore/charts/pull/423
* @pharaujo made their first contribution in https://github.com/kedacore/charts/pull/426

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.10.1...v2.10.2

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.10.2'>KEDA Chart v2.10.2</a>.</em>
